src/3rd_party-static/gmock-1.7.0/src/gmock-spec-builders.cc

```diff
@@ -825,22 +824,28 @@ bool Mock::AsyncVerifyAndClearExpectationsLocked(const int timeout_msec_in)
       } while (true);
     } // mockers iteration
 
+  } // state_map iteration
```

Comment:
@pestOO  This code will be re-factor within task https://adc.luxoft.com/jira/browse/APPLINK-24989